crates/rnote-ui/Cargo.toml

```diff
@@ -58,7 +58,10 @@ url = { workspace = true }
 
 [build-dependencies]
 anyhow = { workspace = true }
-glib-build-tools = { workspace = true }
+glib-build-tools = { workspace = true, optional = true }
+
+[features]
+use_glib_build_tools = ["glib-build-tools"]
```

Comment:
It should be mentioned here that this feature is enabled progammatically in the `build.rs` file and not supposed to be used be the user